### PR TITLE
Feature/integrating u iwith backend

### PR DESCRIPTION
**Pull request by DKusarek**
added project name and spint time frames to Current Sprint View
fixes from PR feature/newfunctonalityguivol2
scrumMaster can see all project
max length of project name is 22 characters
update user Data improved
delete user functionality
enter on loging works

### CODE REVIEW

**Comment by pawelaugustyn** [comment]
Logując się jako gość kafelki z listą projektów są totalnie niewidoczne (dopiero po najechaniu na kafelek cokolwiek da się zobaczyć)

File: ScrumIt/Forms/CurrentSprint.cs [hunk 1]
```diff
@@ -46,6 +46,10 @@ private void CurrentSprint_Load(object sender, EventArgs e)
         {
             try
             {
+                var project = ProjectModel.GetProjectById(_projectId);
+                projectNameTextBox.Text = project.ProjectName;
+                var sprint = SprintModel.GetCurrentSprintForProject(_projectId);
```
[the remark this answers is not in the document]
Comment (salvatien) [reply ScrumIt/Forms/CurrentSprint.cs]: I tested it and it definitely should be changed. If there is no current sprint for the project, everything breaks down, while with GetMostRecent it works just fine

**Comment by salvatien** [comment]
Resolved merge conflicts and changed CurrentSprint_Load so that it uses GetMostRecentSprint for getting the sprint as GetCurrentSprint does not work properly when there is no current sprint at the given moment. Guest's project list issue still present, and the question about optimization holds as well

**Comment by salvatien** [comment]
I broke the tests with adding stuff directly to the database, wait until it is sorted out

**Pull request merged**